Tensile/SolutionStructs.py

```diff
@@ -2605,7 +2605,7 @@ def assignDerivedParameters(state):
           # For SingleBuffer algorithm, _GA and _WorkspaceSizePerElemC is updated only if the gemm function is HPA. The worskspace is
           # used to convert the final output from ComputeDataType to DestDataType. For non-HPA gemm functions the _GA and _Workspace
           # remain unchanged.
-          if computeName != state["ProblemType"]["DataType"].toName(): # for HPA cases
+          if (not state["ProblemType"]["DataType"].isInt8() and computeName != state["ProblemType"]["DataType"].toName()): # for HPA cases
```

Comment:
Why we need "not state["ProblemType"]["DataType"].isInt8()" here?
Int8 type will be rejected later and it will not matter, right?

Reply:
Not really. GSU is valid for int8, both SingleBuffer and MultipleBuffer. For the SB, _GA should be none. That is why we need this.